netlify-plugin-cloudinary/tests/mocks/demo.json

```diff
@@ -1,6 +1,12 @@
 {
   "assets": {
     "images": [
+      {
```

Comment:
this doesnt seem right - why would we want to store an in valid image error like this?